Assets/GothicVR/Scripts/Properties/AbstractProperties.cs

```diff
@@ -4,6 +4,31 @@ namespace GVR.Properties
 {
     public abstract class AbstractProperties : MonoBehaviour
     {
-        public GameObject go => gameObject;
+        private GameObject _customGameObject = null;
```

Comment:
This feels complex. Why do we set a customGO? Please comment at least the use case.